apps/portal/redirects.mjs

```diff
@@ -613,7 +611,7 @@ const walletRedirects = {
     "/connect/wallet/user-management/get-user-profiles",
 
   "/connect/in-app-wallet/overview": "/connect/wallet/overview",
```

Comment:
Lets also update these redirects to go from one /connect/... to another /connect/.. - and directly redirect to /wallets/...